src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -75,6 +97,58 @@ public void archiveListByTitle(final String listTitle) {
         archiveListButton.click();
     }
 
+    /**
+     * Adds a new card to a list.
+     * @param listTitle defines the lists where add a card.
```

Comment:
Please, add white space above this line.

Reply:
Solved